app.sh

```diff
@@ -0,0 +1,25 @@
+# Import data to mysql
+mysql -u root < data.sql
```

Comment:
You'll want to add some way of configuring the username/password/database this is applied to, since not everyone will have the same local configuration. Another way to work around this would be to edit this script to use a mysql docker image, which would have a consistent setup on every run

Reply:
I made the update. Does the new README make sense?